ownership.md

```diff
@@ -0,0 +1,56 @@
+# Production Services, Ownership and Maintenance
+
+For applications, services and tools that we run in "production" (whether that is on developer machines, on a server or in the browser) there is a basic set of requirements that are expected to be met. These requirements exist to ensure consistent maintainability, security, observability and resiliency standards across our estate. 
+
+On top of that, every codebase should have ongoing security and upkeep maintenance applied to it. *This may only be lightweight and relatively infrequent when the project is low priority/risk.*
+
+## Baseline requirements for production services and applications
+
+N.B. This guidance only intended as a minimum baseline; in practice the expectation will be much higher for critical projects.
+
+### CI/CD
+- All source code should be version controlled using [GitHub](./github.md)
+- CI/CD should be employed
+    - An appropriate testing strategy should be considered. We do not aim for a specific % of test coverage, but important business logic should be unit tested
+    - For CI, we use GitHub Actions and [Teamcity](https://teamcity.gutools.co.uk/)
+    - For most (non-library) projects, deployment will be done via [Riff-Raff](https://riffraff.gutools.co.uk/)
+
+### Security
+- A basic [security](./security.md) assessment should be performed to understand the risks and available controls. E.g. 
+authentication, network security, encryption, secret management. Expert guidance from outside the team should sought for high risk applications (e.g. processing user data)
+- Any dependency manifest files should be scanned using [Snyk Open Source](https://security-hq.gutools.co.uk/documentation/snyk)
+- Internal tools should be behind Google Authentication 
+    - A helper exists for [Scala](https://github.com/guardian/play-googleauth) and authentication can be added to an [ALB directly](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/listener-authenticate-users.html)
+    - Network-layer restrictions may also be recommended based on the context
+- All new 3rd party software / vendors used as part of a service should have approval via the [official process](https://spike.gnmremote.com/content/178/requesting-software-for-your-gnm)
+
+### Infrastructure
+- AWS infrastructure should *not* be deployed to the Developer Playground account
```

Comment:
Is it worth saying why too? Is it because the playground account can be cleared at anytime?

Reply:
Plus security concerns. I think we essentially treat the account as a test environment, so adding production services/data would be a concern. Access to the account is also very wide.